dts/bindings/sdhc/cdns,sdhc.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cadence SDHC Controller node
+
+compatible: "cdns,sdhc"
+
+include: [sdhc.yaml, reset-device.yaml]
+
+properties:
+  clock-frequency:
+    type: int
+  reg:
+    required: true
+  power_delay_ms:
```

Comment:
Please add additional documentation for the DT properties you are defining here, at a minimum a `description` value

Reply:
Done.